freshmaker/handlers/internal/update_db_on_odcs_compose_fail.py

```diff
@@ -37,16 +37,19 @@ class UpdateDBOnODCSComposeFail(BaseHandler):
     order = 0
 
     def can_handle(self, event):
-        if not isinstance(event, ODCSComposeStateChangeEvent):
```

Comment:
You need to keep this check before checking compose id, other events may not have compose in event at all.